magprime/templates/magprime_dept_checklist/treasury.html

```diff
@@ -2,9 +2,9 @@
 {% block title %}Department Checklist - Tech Ops{% endblock %}
```

Comment:
The changes in this file were requested by Craig, who noted that this step on the checklist was ONLY about MPoints and therefore anywhere that we say "Cash" should have the word "Cash" removed.